test/router_lorawan_SUITE.erl

```diff
@@ -568,7 +568,7 @@ lw_join_test(Config) ->
         {channel_mask, Mask} ->
             case Region of
                 'US915' ->
-                    ExpectedMask = lists:seq(8, 15),
+                    ExpectedMask = lists:seq(0, 7),
                     Mask = ExpectedMask;
```

Comment:
Not that you touched this line, but for sanity when debugging this test case in future it would be useful to see both values in test output instead of `bad match` or whatever.

```suggestion
                    ?assertEqual(ExpectedMask, Mask);
```